lib/index.js

```diff
@@ -74,10 +76,20 @@ export default function rehypePicture(options) {
 
       for (key in map) {
         if (Object.hasOwn(map, key)) {
+          /** @type {{srcSet: string, type: string | null | undefined, width?: string, height?: string}} */
+          const prop = {srcSet: replaceExt(src, '.' + key), type: map[key]}
+          if (width) {
+            prop.width = String(width)
+          }
+
+          if (height) {
+            prop.height = String(height)
+          }
```

Comment:
```suggestion
          const properties = {
            srcSet: replaceExt(src, '.' + key),
            type: map[key]
          }

          if (width) {
            properties.width = width
          }

          if (height) {
            properties.height = height
          }
```

No need to cast to string, and lets give this a full name: `properties`